adapter/stackdriver/stackdriver.go

```diff
@@ -17,7 +17,7 @@ package stackdriver
 import (
 	"context"
 
-	multierror "github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-multierror"
```

Comment:
small comment style-wise for your consideration: since the package path is `go-multierror` but it is actually the package `multierror`, using an alias to make that explicit in the client code can help improve readability. 

up to you.

Reply:
goimports removed it (or maybe gogland did), IDC either way.

Reply:
Heh, I added it back in my commit but it was removed when I formatted pre-commit. So definitely goimports removing it.

Reply:
interesting. i don't feel strongly at all. proceed.